src/pages/RepoPage/BundlesTab/BundleContent/AssetsTable/useBundleAssetsTable.ts

```diff
@@ -25,6 +25,8 @@ export function useBundleAssetsTable({
 
   // @ts-expect-error - useLocationParams needs fixing
   const typeFilters = params?.types ?? []
+  // @ts-expect-error - useLocationParams needs fixing
```

Comment:
Will TS router fix this? Or will we still need to use useLocationParams?

Reply:
The new router will solve this, we will also have query param validation ensuring that we don't override parent params from child routes.